default.nix

```diff
@@ -12,7 +12,7 @@
 
   # Mixins
   mixins-cloud-init = import ./mixins/cloud-init.nix;
-  mixins-efi = import ./mixins/efi.nix;
+  mixins-systemd-boot = import ./mixins/systemd-boot.nix;
   mixins-nginx = import ./mixins/nginx.nix;
   mixins-telegraf = import ./mixins/telegraf.nix;
```

Comment:
It's a good habit to also re-order alphabetically when renaming things

Reply:
It's just not easy to maintain with a pull request based workflow and without automation.